_P152_MCP42010.ino

```diff
@@ -29,7 +27,7 @@ boolean Plugin_152(byte function, struct EventStruct *event, String& string)
     case PLUGIN_DEVICE_ADD:
       {
         Device[++deviceCount].Number = PLUGIN_ID_152;
-        Device[deviceCount].Type = DEVICE_TYPE_DUMMY;           // SPI pins for ESP8266 are CS=15, CLK=14, MOSI=13
+        Device[deviceCount].Type = DEVICE_TYPE_SPI;
```

Comment:
If that still works for you, you are using a _very_ old source release for ESPEasy...